libs/features/pages/booklets-page/src/lib/booklets-page.module.ts

```diff
@@ -1,10 +1,16 @@
 import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
+
 import { BookletsPageComponent } from './main/booklets-page/booklets-page.component';
+import { BookletsHeroSectionComponent } from './components/booklets-hero-section/booklets-hero-section.component';
+import { AppHeaderModule } from '@elewa-website/elements/layout/header';
+import { TextsModule } from '@elewa-website/elements/layout/texts';
+import { ButtonsModule } from '@elewa-website/elements/layout/buttons';
+
 import { BookletsRoutingModule } from './booklets.routing';
 
```

Comment:
Kindly, check the order of imports. The elements imports should appear before local imports

Reply:
@AnnaL001 I have made the changes